.travis.yml

```diff
@@ -16,12 +16,12 @@ install:
 
 script:
 - make generate
-- if [[ -n "$(git status --porcelain)" ]] ; then echo "It seems like you need to run
+- if [[ -n "$(git status --porcelain)" && $TRAVIS_CPU_ARCH == "amd64" ]] ; then echo "It seems like you need to run
```

Comment:
I see that you wrote that this causes differences. That is explicitly not what we want. What are the differences? They should result in predictable results.

Reply:
looks like it creates files in cluster-up that it doesnt need to
its not the current problem maybe, but generally can cause headache i think ?
cluster-up/cluster/kind-k8s-sriov-1.17.0/csrcreator/BUILD.bazel

Reply:
This is suspicious, yes.